### DISCUSSION TITLE
How to change the iframe link

### DISCUSSION THREAD

**Discussion by RakulAgn** [GrapesJS/grapesjs#5675] [General]
<img width="496" alt="image" src="https://github.com/GrapesJS/grapesjs/assets/119593989/6f2f13f0-cd7e-456c-b306-dded2a954e37">

is that possible to change the iframe link

**Comment by artf** [comment]
why?

**Reply by RakulAgn** [reply]
I have a feature to render users given site link behind the iframe or i need to take the screenshot of the page to set the background image to the iframe

**Reply by artf** [reply]
Just create your own iframe and place it into the canvas, no need to break the internal one.
